app/src/lib/stores/git-store.ts

```diff
@@ -544,7 +544,7 @@ export class GitStore extends BaseStore {
       }
     }
 
-    this._contextualCommitMessage = {
+    this._commitMessage = {
```

Comment:
With these changes do we have any actual need for `_contextualCommitMessage`?  The only context I can see being done in `GitStore` is loading the merge message, which was part of our old merge conflicts flow. If there's no need for this now that you're favouring `commitMessage` then I think we can clean this up from the stores and components that need it

Reply:
ooo good catch! i will 🔥it!